src/qtiCommonRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -79,7 +79,9 @@ function render(interaction) {
          * @param {jQueryElement} $container   - container element to adapt
          */
         const resize = _.debounce(() => {
-            if (interaction.mediaElement) {
+            // only resize when width in px
+            // new version has width in %
+            if (interaction.mediaElement && media.attr('width') && !/%/.test(media.attr('width'))) {
```

Comment:
Regexp is slow. `includes` is better, but if you need ie11 support, you should use `indexOf`
```suggestion
            let width = media.attr('width');
            if (interaction.mediaElement && width && !width.includes('%'))) {
```